src/editorialreview/tests.py

```diff
@@ -0,0 +1,93 @@
+import factory
+
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.test.client import Client
+
+from editorialreview import models
+from core import models as core_models
+from review import models as review_models
+from submission import models as submission_models
+
+
+class UserFactory(factory.Factory):
```

Comment:
I like the idea of having all factories living in a dedicated factory file in the test module, so you can import them like this:

```
from editorialreview.test.factories import *
```

But this is allright for now